examples/install_simple/main.tf

```diff
@@ -14,9 +14,30 @@
  * limitations under the License.
  */
 
+provider "google" {
```

Comment:
I really don't understand why GitHub is showing these provider changes as being added, they already exist in master branch: https://github.com/forseti-security/terraform-google-forseti/blob/master/examples/install_simple/main.tf#L17